README.md

```diff
@@ -285,23 +287,25 @@ view emp_v departments employees
 
         <script>
             new quickERD.Diagram( quickSQL.toERD(
-`departments /insert 2
+`
+departments
     name /nn
     location
     country
-    employees /insert 4
-        name /nn vc50
-        email /lower
-        cost center num
-        date hired
-        job vc255
+employees
+    departments_id /fk departments
```

Comment:
Same as above, why explicitly define the FK where you can just indent the table?